merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/http/HasuraParsers.java

```diff
@@ -54,6 +55,17 @@ private static <I extends HasuraAction.Input> JsonParser<HasuraAction<I>> hasura
                           .field("planId", planIdP)
                           .map(HasuraAction.PlanInput::new, HasuraAction.PlanInput::planId));
 
+  public static final JsonParser<HasuraAction<HasuraAction.SimulateInput>> hasuraSimulateActionP
+      = hasuraActionF(
+          productP
+              .field("planId", planIdP)
+              .optionalField("force", nullableP(boolP))
+              .map(
+                  untuple((planId, force) -> new HasuraAction.SimulateInput(planId, force.flatMap($ -> $))),
+                  $ -> tuple($.planId(), Optional.of($.force()))
```

Comment:
Non-blocker, but it might be clearer to just have this line be the `$.orElse(false)` so we can propagate a `Boolean` down to `getSimulationResults` instead of an `Optional<Boolean>`

Reply:
I've addressed this slightly differently. Rather than update the parser, I've instead updated when `force` is fetched from `body` in `MerlinBindings` to flatten it to a boolean as you suggested.

Reply:
Looks great!